app/src/main/java/org/eyeseetea/malariacare/domain/usecase/GetSurveyAnsweredRatioUseCase.java

```diff
@@ -3,17 +3,17 @@
 import org.eyeseetea.malariacare.data.database.model.ProgramDB;
 import org.eyeseetea.malariacare.data.database.model.QuestionDB;
 import org.eyeseetea.malariacare.data.database.model.SurveyDB;
+import org.eyeseetea.malariacare.data.database.model.SurveyAnsweredRatioDB;
 import org.eyeseetea.malariacare.data.database.model.ValueDB;
 import org.eyeseetea.malariacare.domain.entity.SurveyAnsweredRatio;
-import org.eyeseetea.malariacare.domain.entity.SurveyAnsweredRatioCache;
 
 public class GetSurveyAnsweredRatioUseCase {
     public interface Callback{
         void nextProgressMessage();
         void onComplete(SurveyAnsweredRatio surveyAnsweredRatio);
     }
 
-    public enum RecoveryFrom {DATABASE, MEMORY_FIRST }
+    public enum Action {FORCE_UPDATE, GET}
```

Comment:
We should not couple get and save AnsweredRatio processes, this use case should only get from database and we should create a SaveAnsweredRatioUseCase